src/test/skript/tests/syntaxes/effects/EffLoop.sk

```diff
@@ -0,0 +1,15 @@
+test "inline loop times":
+	do add loop-value to {_sum} 5 times
+	assert {_sum} is (1 + 2 + 3 + 4 + 5) with "inline loop times failed"
+
+test "inline loop foreach":
+	spawn a cow at spawn of "world"
+	set {_first} to last spawned cow
+	spawn a cow at spawn of "world"
+	set {_second} to last spawned cow
+	do add uuid of loop-entity to {_uuids::*} for every element in all cows
+	assert size of {_uuids::*} is 2 with "inline loop foreach failed"
+	loop {_uuids::*}:
+		assert loop-value is (uuid of {_first}) or (uuid of {_second}) with "inline loop foreach content failed"
+	delete {_first}
+	delete {_second}
```

Comment:
Doing `delete {_var}` here shouldn't delete the entity, should change it to `delete entities`

Reply:
![image](https://user-images.githubusercontent.com/25848425/207979297-9227b0a1-0a47-4447-9232-092ca695a1fc.png)
Should delete the entity

Reply:
from testing in game I don't believe it does
![image](https://user-images.githubusercontent.com/81952476/207979729-2e973010-23b9-466d-83c6-88f8d3809106.png)


it's been like this forever so unless it's a test run only thing it doesn't

Reply:
there is a pr for it just not exactly like it
https://github.com/SkriptLang/Skript/pull/5198